src_assets/common/assets/web/configs/tabs/Advanced.vue

```diff
@@ -73,6 +73,7 @@ const config = ref(props.config)
           <template #windows>
             <option value="ddx">Desktop Duplication API</option>
             <option value="wgc">Windows.Graphics.Capture {{ $t('_common.beta') }}</option>
+            <option value="amd">AMD Display Capture {{ $t('_common.beta') }}</option>
```

Comment:
Let's order these in the same order that they are attempted.

Does the AMD capture work with the Windows service? We only marked WGC beta because it doesn't... we can still mark the AMD one as beta if there are other still known limitations when this gets merged.